examples/gpu/shard-cpu-worker.config.example

```diff
@@ -0,0 +1,241 @@
+# the listening port of the shard-worker grpc server
```

Comment:
Drop this file and others in gpu, include comments in shard-worker example at base